src/components/MyList/MyList.tsx

```diff
@@ -0,0 +1,34 @@
+import { memo } from 'react';
+import { User } from '../../models/user';
+import { MyListItem } from '../MyListItem/MyListItem';
+import './MyList.css';
+
+interface MyListProps {
+  users: User[];
+}
+
+function MyList(props: MyListProps) {
+  const { users } = props;
+
+  console.log('render list');
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td className="header">Is Active</td>
+          <td className="header">Name</td>
+          <td className="header">Age</td>
+          <td className="header">Organization</td>
+        </tr>
+      </thead>
+      <tbody>
+        {users.map((user, index) => {
+          const isLast = index === users.length - 1;
```

Comment:
nit: you can detect the last row with CSS selectors only, no need for runtime javascript here

Reply:
done